.editorconfig

```diff
@@ -12,6 +12,5 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = true
```

Comment:
kind of funny that this file doesn't have a new line at the end 😆

Reply:
Dude! I thought that was hilarious. I was literally expecting it to happen as soon as I hit safe...so disappointed. 🤣